src/api/handlers/webhook.ts

```diff
@@ -552,5 +553,10 @@ export async function editSlashResponse(
     options,
   );
 
-  return result;
+  if (options.messageID) return result;
```

Comment:
`Edit Webhook Message` does return a message object too, so why not using `structures.createMessage` here too?

Reply:
Good catch that should have been if (!options.messageID